_i18n/en/resources/user-guides/restore_account.md

```diff
@@ -1,53 +1,131 @@
+- [When should I restore a wallet?](#when-should-i-restore-a-wallet)
+- [What is needed to restore a wallet from mnemonic seed?](#what-is-needed-to-restore-a-wallet-from-mnemonic-seed)
+- [How to restore a wallet from mnemonic seed on Monero GUI](#how-to-restore-a-wallet-from-mnemonic-seed-on-monero-gui)
+- [How to restore a wallet from mnemonic seed on Monero CLI](#how-to-restore-a-wallet-from-mnemonic-seed-on-monero-cli)
+- [My restored wallet is showing a wrong balance](#my-restored-wallet-is-showing-a-wrong-balance)
 
-### Account Software:  monero-wallet-cli
+### When should I restore a wallet?
 
-- Open a command prompt and navigate to the drive and directory that contains monero-wallet-cli
+You should restore your @wallet in the following situations:
+- You lost access to your wallet (you forgot its password, your wallet file is corrupted or your device stopped working/is lost).
+- You want to create a copy of your wallet on another device.
 
-- At the command prompt type:  `monero-wallet-cli --restore-deterministic-wallet`
+This guide will teach you how to restore a Monero wallet from its @mnemonic-seed.
 
-- Once you press enter you will be prompted for a wallet file name.  Give your wallet a new name, any name will do
+If you want to restore a wallet from its private keys, please read [this guide]({{site.baseurl}}/resources/user-guides/restore_from_keys.html).
 
-- Press enter again and you will be prompted for a password.  Give your wallet a new and long password
+If you want to restore a wallet from a hardware wallet, please check [this page]({{site.baseurl}}/downloads/#hardware).
 
-- Press enter again you will be prompted to repeat the password
+### What is needed to restore a wallet from mnemonic seed?
 
-- Press enter again and you will be prompted for the 25 word electrum style mnemonic seed that you retrieved earlier
+In order to restore a wallet from mnemonic seed, you must have a valid Monero mnemonic seed. A Monero mnemonic seed is a phrase with 13, 14, or 25 random words (multiple languages are supported). There are three types of Monero mnemonic seed:
 
--  You will then be prompted with "Restore from specific blockchain height (optional, default 0):" Default will start the restore process from the beginning of the Monero blockchain. If you don't know the specific blockchain height, just hit enter. (Specifying a specific blockchain height will start the restore process from that specific height. This will save a bit of time in scanning, if you know what starting blockchain height your initial funds were transacted for this specific account.)
+- **25 words mnemonic seed**: used by most Monero wallets and supported by all Monero wallets.
+- **14 words mnemonic seed with embedded restore height**: used and supported by Feather wallet.
+- **13 words mnemonic seed**: deprecated, previously used by MyMonero wallet. Supported by most Monero wallets.
```

Comment:
This could be much shorter. IMO it's not necessary to add a dedicated explanation of the various type of seeds. You already mention above that there are more than one.